src/goto-analyzer/static_verifier.h

```diff
@@ -23,4 +25,9 @@ bool static_verifier(
   message_handlert &,
   std::ostream &);
 
+void static_verifier(
```

Comment:
This should be documented. And it's a bit confusing to use a noun for a function which is returning void. (for me "verifier"    here should imply you return an object which has a `verify` method)

Reply:
This is doing more or less the same thing as the other `static_verifier` function that already is in this file, the only difference is that it writes the result to the `propertiest` passed in rather than writing them to stdout or to a file.